gpii/node_modules/flowManager/src/BrowserChannel.js

```diff
@@ -0,0 +1,37 @@
+(function () {
+
+    "use strict";
+
+    var fluid = require("infusion");
+    var gpii = fluid.registerNamespace("gpii");
+
+    fluid.registerNamespace("gpii.request.flowManager");
+
+    fluid.defaults("gpii.flowManager.browserChannel", {
+        gradeNames: ["fluid.standardRelayComponent", "autoInit"]
+    });
+
+    fluid.defaults("kettle.requests.request.handler.browserChannel", {
+        gradeNames: ["gpii.request.flowManager.sessionAware", "autoInit"],
+        invokers: {
+            handle: {
+                funcName: "gpii.request.flowManager.browserChannelHandle",
+                args: ["{request}"],
+                dynamic: true
+            }
+        }
+    });
+
+    gpii.request.flowManager.browserChannelHandle = function (fluidRequest) {
+        var data = fluidRequest.data;
```

Comment:
> Should make some attempt to validate the data here. It should correspond to something in the solutions registry or at least be a string of the appropriate form - otherwise we are subject to obvious DoS attacks

@amb26, 

I have added a check in [this commit](https://github.com/javihernandez/universal/commit/05b93274e9d23c552610b61048db5a8641450e4b). Although the client won't receive any messages from the webSockets settings handler, it's still "connected" to the system. How can I _break_ this connection? Is there any way to do this, or should I rely on the client to close the connection?